shared/utils.ts

```diff
@@ -1,5 +1,12 @@
+import StoreService from '../desktop/storeService';
+
 // eslint-disable-next-line import/prefer-default-export
 export const delay = (ms: number) =>
   new Promise<void>((resolve) => {
     setTimeout(() => resolve(), ms);
   });
+
+export const setTheme = (key: string, value: any) => {
+  StoreService.init();
```

Comment:
We still have two instances of the persistent store, which may cause bugs.
Moreover, we should not work with `fs` directly from the renderer process (if it works at all without a workaround).

So I propose to:
- use persistent store only in the main process (init once)
- push the value of persistent store to the redux store (main -> renderer)
- update persistent store on any change in redux store (renderer -> main)
- update redux store on any update in a persistent store (main -> renderer)

To provide syncing functionality I propose to write a small redux-middleware, that will take care of two-way communications (IPC) and syncing states.